File: sample/src/main/java/com/pinger/repertory/recycler/RecyclerViewActivity.java
package com.pinger.repertory.recycler;

import com.pinger.baselib.base.BaseActivity;
import com.pinger.repertory.R;

/**
 * @author Pinger
 * @since 2018/1/13 0013 上午 11:11
 */

public class RecyclerViewActivity extends BaseActivity {

    @Override
    protected int getLayoutResID() {
        return R.layout.activity_recycler_view;
    }

    @Override
    protected void initView() {
        setActionBar(getString(R.string.recycler_view),true);
    }
}

File: sample/src/main/java/com/leibo/repertory/recycler/RecyclerViewActivity.kt
package com.leibo.repertory.recycler

import com.leibo.baselib.base.basic.BaseActivity
import com.leibo.repertory.R
import java.util.function.Consumer

/**
 * @author Pinger
 * @since 2018/1/13 0013 上午 11:11
 */

class RecyclerViewActivity : BaseActivity() {

    override val layoutResID: Int
        get() = R.layout.activity_recycler_view

    override fun initView() {
        setActionBar(getString(R.string.recycler_view), true)
    }

    override fun initData() {
        val url = ""

    }
}
